### ISSUE TITLE
Following the "Using THREE from npm" code causes a `THREE is not defined` error

### ISSUE THREAD

**Issue by raray** [tengbao/vanta#69]
My code:
```
import * as THREE from 'three'
import BIRDS from 'vanta/dist/vanta.birds.min'
 
document.addEventListener('DOMContentLoaded', (event) => { 
  BIRDS({
    el: '#test',
    THREE: THREE
  })
})
```

Results in the following errors:
```
[vanta.js] birds init error:  ReferenceError: THREE is not defined
    at new a (vanta.birds.min.js:1)
    at p.initComputeRenderer (vanta.birds.min.js:1)
    at p.onInit (vanta.birds.min.js:1)
    at p.init (vanta.birds.min.js:1)
    at new o.VantaBase (vanta.birds.min.js:1)
    at new p (vanta.birds.min.js:1)
    at o.<computed> (vanta.birds.min.js:1)
    at HTMLDocument.<anonymous> (prism.js:5)

vanta.birds.min.js:1 Uncaught TypeError: Cannot read property 'time' of undefined
    at p.onUpdate (vanta.birds.min.js:1)
    at p.animationLoop (vanta.birds.min.js:1)
    at new o.VantaBase (vanta.birds.min.js:1)
    at new p (vanta.birds.min.js:1)
    at o.<computed> (vanta.birds.min.js:1)
    at HTMLDocument.<anonymous> (prism.js:5)
```

`THREE` is seen a defined if logged out before calling `BIRDS`:
```
{ACESFilmicToneMapping: 4, AddEquation: 100, AddOperation: 2, AdditiveAnimationBlendMode: 2501, AdditiveBlending: 2, …}
```

**Comment by raray** [comment]
As far as I can tell, the `THREE` that is passed in to the constructor isn't reassigned before it is used here:
```
var n = new THREE.Scene, o = new THREE.Camera;
```

**Comment by tengbao** [comment]
Hmm, can you please try the latest version here? https://github.com/tengbao/vanta/blob/master/dist/vanta.birds.min.js @raray

**Comment by tengbao** [comment]
Confirmed fixed in 0.5.17
